lib/pages/home/profile_card.dart

```diff
@@ -28,13 +30,29 @@ class ProfileCard extends StatelessWidget {
               Row(
                 crossAxisAlignment: CrossAxisAlignment.center,
                 children: <Widget>[
-                  const Padding(
-                    padding: EdgeInsets.symmetric(horizontal: 8),
+                  Padding(
+                    padding: const EdgeInsets.symmetric(horizontal: 8),
                     child: CircleAvatar(
                       radius: 40,
-                      child: Image(
-                          image: AssetImage(
-                              'assets/illustrations/undraw_profile_pic.png')),
+                      child: user != null
+                          ? FutureBuilder(
+                              future: StorageProvider.findImageUrl(context,
```

Comment:
If "upload profile picture" is an action done by the authProvider, so should "get profile picture URL" be. The UI shouldn't know about where the picture is stored.